README.md

```diff
@@ -189,6 +189,44 @@ class MyOtherModel(models.Model):
         return cls.objects.filter(users__count__gt=0, owner__is_active=True)
 ```
 
+## Using instance metadata
+
+You can leverage the `CRUDChange.metadata` JSON field when you want to save some other related data to each change. For example when aggregating the changes across multiple models by some specific business logic.
+To populate this field, you should declare a method on the model that requires it.
+
+The default method name is `get_easyaudit_metadata`, but it can be configured by adding the `EASY_AUDIT_METADATA_METHOD`field on the model.
```

Comment:
```suggestion
The default method name is `get_easyaudit_metadata`, but it can be configured by defining the `EASY_AUDIT_METADATA_METHOD` field on the model.
```